README.md

```diff
@@ -1 +1,36 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+
+> This is the first milestone in the process of creating my portfolio website.
+
+![screenshot](Images/Screenshot.png)
+
+This website is a setup and mobile version skeleton of my portfolio.
+
+## Built With
+
+- HTML
+- CSS
+- VSCode
+
+
+## Authors
+
+👤 **Sahar Abdel Samad**
+
+- GitHub: [@sahar-abdelsamad](https://github.com/Sahar-AbdelSamad)
+- Twitter: [@abdelsamadsahar](https://twitter.com/home)
+- LinkedIn: [abdel-samad-sahar](https://www.linkedin.com/in/abdel-samad-sahar-353977223/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [OPTIONAL] Kindly add issues page link of your GitHub repository

Reply:
Thank you for the information.